.github/workflows/scripts/code_coverage_disable_check.py

```
@@ -0,0 +1,161 @@
+
+    Returns:
+        bool: True if code coverage disable statement is found, False
+        otherwise.
+    """
+    code_coverage_disable_pattern = re.compile(
+        r"""//?\s*istanbul\s+ignore(?:\s+(?:next|-line))?[^\n]*|
+        /\*\s*istanbul\s+ignore\s+(?:next|-line)\s*\*/""",
+        re.IGNORECASE,
+    )
+    try:
+        with open(file_path, "r", encoding="utf-8") as file:
```

Ruff (0.8.2)

Unnecessary open mode parameters

Remove open mode parameters

(UP015)